crates/astria-sequencer/src/fees/storage/values.rs

```diff
@@ -29,178 +32,81 @@ pub(crate) struct Value(ValueImpl);
-}
-
-#[derive(Debug, BorshSerialize, BorshDeserialize)]
-pub(in crate::fees) struct IbcRelayerChangeFeeComponentsStorage {
+pub(in crate::fees) struct FeeComponents {
     pub base: u128,
     pub multiplier: u128,
 }
 
-#[derive(Debug, BorshSerialize, BorshDeserialize)]
-pub(in crate::fees) struct IbcSudoChangeFeeComponentsStorage {
-    pub base: u128,
-    pub multiplier: u128,
+impl<T> From<DomainFeeComponents<T>> for FeeComponents {
+    fn from(fees: DomainFeeComponents<T>) -> Self {
+        Self {
+            base: fees.base(),
+            multiplier: fees.multiplier(),
+        }
+    }
 }
 
-#[derive(Debug, BorshSerialize, BorshDeserialize)]
-pub(in crate::fees) struct SudoAddressChangeFeeComponentsStorage {
-    pub base: u128,
-    pub multiplier: u128,
+impl<T: ?Sized> From<FeeComponents> for DomainFeeComponents<T> {
+    fn from(fees: FeeComponents) -> Self {
+        Self::new(fees.base, fees.multiplier)
+    }
 }
 
-impl_from_for_fee_component!(
-    TransferFeeComponents => TransferFeeComponentsStorage,
-    RollupDataSubmissionFeeComponents => RollupDataSubmissionFeeComponentsStorage,
-    Ics20WithdrawalFeeComponents => Ics20WithdrawalFeeComponentsStorage,
-    InitBridgeAccountFeeComponents => InitBridgeAccountFeeComponentsStorage,
-    BridgeLockFeeComponents => BridgeLockFeeComponentsStorage,
-    BridgeUnlockFeeComponents => BridgeUnlockFeeComponentsStorage,
-    BridgeSudoChangeFeeComponents => BridgeSudoChangeFeeComponentsStorage,
-    IbcRelayFeeComponents => IbcRelayFeeComponentsStorage,
-    ValidatorUpdateFeeComponents => ValidatorUpdateFeeComponentsStorage,
-    FeeAssetChangeFeeComponents => FeeAssetChangeFeeComponentsStorage,
-    FeeChangeFeeComponents => FeeChangeFeeComponentsStorage,
-    IbcRelayerChangeFeeComponents => IbcRelayerChangeFeeComponentsStorage,
-    IbcSudoChangeFeeComponents => IbcSudoChangeFeeComponentsStorage,
-    SudoAddressChangeFeeComponents => SudoAddressChangeFeeComponentsStorage,
-);
-
 impl_from_for_fee_storage!(
-    TransferFeeComponentsStorage => TransferFees,
-    RollupDataSubmissionFeeComponentsStorage => SequenceFees,
-    Ics20WithdrawalFeeComponentsStorage => Ics20WithdrawalFees,
-    InitBridgeAccountFeeComponentsStorage => InitBridgeAccountFees,
-    BridgeLockFeeComponentsStorage => BridgeLockFees,
-    BridgeUnlockFeeComponentsStorage => BridgeUnlockFees,
-    BridgeSudoChangeFeeComponentsStorage => BridgeSudoChangeFees,
-    IbcRelayFeeComponentsStorage => IbcRelayFees,
-    ValidatorUpdateFeeComponentsStorage => ValidatorUpdateFees,
-    FeeAssetChangeFeeComponentsStorage => FeeAssetChangeFees,
-    FeeChangeFeeComponentsStorage => FeeChangeFees,
-    IbcRelayerChangeFeeComponentsStorage => IbcRelayerChangeFees,
-    IbcSudoChangeFeeComponentsStorage => IbcSudoChangeFees,
-    SudoAddressChangeFeeComponentsStorage => SudoAddressChangeFees,
+    DomainFeeComponents<Transfer> => TransferFees,
+    DomainFeeComponents<RollupDataSubmission> => SequenceFees,
```

Comment:
```suggestion
    DomainFeeComponents<RollupDataSubmission> => RollupDataSubmissionFees,
```

Reply:
Done in [2a3ab67](https://github.com/astriaorg/astria/pull/1811/commits/2a3ab67b689c045ab5bb0ee99f705f0a04ad9437).